ifcUtils/repo_ifc_utils_tree_parser.cpp

```diff
@@ -1268,7 +1269,7 @@ void  repo::ifcUtility::SCHEMA_NS::TreeParser::generateClassificationInformation
 				if (refSource->hasSource()) {
 #endif
 					classificationName = refSource->Name();
-					metaValues[constructMetadataLabel("Name", classificationName)] = classificationName;
+					metaValues[constructMetadataLabel("Name", classificationName)] = classificationName.c_str();
```

Comment:
same here

Reply:
c_str() is not needed it was done for some testing purpose but should be removed, hence removing it now.